submissions/Sveti-Stefan/hooli-style-popup/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="container">
+            <header class="container_header">
+                <div class="logo_header">
+                    <img src="svg/logo.svg" alt="Logo">
+                </div>
+                <nav class="menu_header_nav">
+                    <ul class="menu_header_list">
+                        <li class="menu_header_list_item"><a href="#">Platform</a></li>
+                        <li class="menu_header_list_item"><a href="#">Support</a></li>
+                        <li class="menu_header_list_item"><a href="#">Company</a></li>
+                        <li class="menu_header_list_item popup_item">
+
+                            <input type="checkbox" id="popup-checkbox" class="popup_checkbox">
+                            <label for="popup-checkbox">
+                                <img src="svg/app.svg" alt="popup-button" class="popup_icon">
+                            </label>
+
+                            <div class="popup">
+                                <ul class="popup_item_list">
+                                    <li><a href="#"><img src="svg/placement.svg"
+                                                alt="Placement"><span>Placement</span></a>
+                                    </li>
+                                    <li><a href="#"><img src="svg/booking.svg" alt="Booking"><span>Booking</span></a>
+                                    </li>
+                                    <li><a href="#"><img src="svg/massenger.svg"
+                                                alt="Massenger"><span>Massenger</span></a>
+                                    </li>
+                                    <li><a href="#"><img src="svg/rewevs.svg" alt="Reviews"><span>Reviews</span></a>
+                                    </li>
+                                    <li><a href="#"><img src="svg/contact.svg" alt="Contact"><span>Contact</span></a>
+                                    </li>
+                                    <li><a href="#"><img src="svg/Sites.svg" alt="Sites"><span>Sites</span></a></li>
+                                </ul>
+
+                                <input type="checkbox" id="popup_checkbox_more" class="popup_checkbox_more">
+                                <label class="more_label" for="popup_checkbox_more">More</label>

```

Comment:
You could add another label for collapsing the list if you like. Technically, you can use multiple labels for an input element.

Reply:
Thanks, I'll keep it in mind.